src/main/java/teammates/logic/core/InstructorsLogic.java

```diff
@@ -250,7 +250,8 @@ public MimeMessage sendRegistrationInviteToInstructor(String courseId, String in
         CourseAttributes course = coursesLogic.getCourse(courseId);
         if (course == null) {
             throw new EntityDoesNotExistException(
-                    "Course does not exist [" + courseId + "], trying to send invite email to student [" + instructorEmail + "]");
+                    "Course does not exist [" + courseId + "], "
+                    + "trying to send invite email to student [" + instructorEmail + "]");
```

Comment:
Not sure why it says student here :laughing: maybe we could open an issue to change this to `instructor` instead if there's a need to change it.